src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -265,12 +278,14 @@ def prepare(self):
             'ceph.crush_device_class': crush_device_class,
         }
         if self.args.filestore:
+            #TODO: allow auto creation of journal on passed device, only works
+            # when physical device is passed, not LV
             if not self.args.journal:
                 raise RuntimeError('--journal is required when using --filestore')
 
             data_lv = self.get_lv(self.args.data)
             if not data_lv:
-                data_lv = self.prepare_device(self.args.data, 'data', cluster_fsid, osd_fsid)
+                data_lv = self.prepare_device(self.args.data, 'data', osd_fsid)
```

Comment:
So, `osd_fsid` is actually the uuid of the OSD?

Reply:
fsid and uuid are used interchangeably throughout the code base. I'd like to unify this towards uuid but that will be a longer process.